app/models/claim.rb

```diff
@@ -211,6 +211,10 @@ class Claim < ApplicationRecord
   scope :not_awaiting_qa, -> { approved.where("qa_required = false OR (qa_required = true AND qa_completed_at IS NOT NULL)") }
   scope :awaiting_qa, -> { approved.qa_required.where(qa_completed_at: nil) }
   scope :qa_required, -> { where(qa_required: true) }
+  scope :awaiting_fe_provider_verification, -> do
```

Comment:
I realise this isn't the only example, but I'd like to avoid using acronyms wherever possible.
```suggestion
  scope :awaiting_further_education_provider_verification, -> do
```